metricbeat/module/ceph/cluster_disk/_meta/fields.yml

```diff
@@ -0,0 +1,20 @@
+- name: cluster_disk
+  type: group
+  description: >
+    cluster_disk
+  fields:
+    - name: stats.available.bytes
```

Comment:
It's kind of strange now that everything is under `stats`. Should we move everything one level up or do we expect other things to come in in the future?